lib/hyku_addons/engine.rb

```diff
@@ -49,17 +49,26 @@ class Engine < ::Rails::Engine
       Hyku::RegistrationsController.class_eval do
         def new
           return super if current_account&.allow_signup == "true"
-          redirect_to root_path, alert: t(:'hyku.account.signup_disabled')
+
+          redirect_to root_path, alert: t("hyku.account.signup_disabled")
         end
 
         def create
           return super if current_account&.allow_signup == "true"
-          redirect_to root_path, alert: t(:'hyku.account.signup_disabled')
+
+          redirect_to root_path, alert: t("hyku.account.signup_disabled")
         end
 
         def current_account
           Site.account
         end
+
+        protected
+
+          # Redirect the user to the profile after registration
+          def after_sign_up_path_for(resource)
```

Comment:
Where is this used?

Reply:
Its part of Devise. https://github.com/heartcombo/devise/wiki/How-To:-Redirect-to-a-specific-page-on-successful-sign-up-%28registration%29